docs/Table.md

```diff
@@ -17,43 +15,70 @@ const { query } = initialize({ iotsitewiseClient });
 
 // jsx
 <Table
-  viewport={{ duration: '5m' }}
+  viewport = {{ duration: '5m' }}
```

Comment:
remove spacing, not normal syntax for JSX to have spaces around the attribute declarations

Reply:
Removed spaces.